.prettierignore

```diff
@@ -2,3 +2,4 @@
 bench/
 coverage/
 node_modules/
+target/
```

Comment:
Did you mean `wasm-mappings/target/`?

You may ignore the whole `wasm-mappings/` as there is also json files being pretty printed in `wasm-mappings/source-map-mappings-wasm-api/`.

Reply:
Without a leading `/`, this'll match any directory named `target`, so it'll take care of `wasm-mappings/source-map-mappings-wasm-api/target/` as well.